src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/PartiallyDeterminedHaplotypeComputationEngine.java

```diff
@@ -903,7 +910,12 @@ private static double dragenStart(final Event event) {
         return event.getStart() + (event.isIndel() ? (event.isSimpleDeletion() ? 1 : 0.5) : 0);
     }
 
-    private static void eventGroupsMessage(final Haplotype referenceHaplotype, final boolean debug, final Map<Double, List<Event>> eventsByDRAGENCoordinates) {
+    private static final boolean dragenOverlap(final Event event1, final Event event2) {
```

Comment:
javadoc this to be explicit about what and why the boundaries are here (for example we allow compound events here) also rename it

Reply:
moot due to previous PR